native/src/components/PoisBottomSheet.tsx

```diff
@@ -92,6 +92,8 @@ const PoisBottomSheet = ({
       enableContentPanningGesture={enableContentPanningGesture}
       enableDynamicSizing={false}
       animateOnMount
+      accessibilityRole={'list'}
+      accessibilityLabel={t('nearYou', { count: pois.length, nearYou: t('listTitle') })}
```

Comment:
:wrench: Since the bottom sheet might contain either a list or a single poi, this accessibility role and label is only correct in the former.